### PR TITLE
Simulation config

### PR DESCRIPTION
**Pull request by JennaGustafson**
Updates to executing simulations, considering the use of different FFs and unit cells

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,4 @@ launch_workers_qsub.sh
 test*
 *csv
 *pyc
+example/*
```
Comment (paulboone) [inline .gitignore:22]: What do you keep in the example dir?

**Review by paulboone — APPROVED** [review]

**Comment by paulboone** [comment]
Commits are organized and named clearly so I can tell what you've done; PR was super easy to review. Thanks! LGTM.

**Comment by JennaGustafson** [comment]
Example has csv files with 1 mof/a few gas mixtures for testing the
simulation scripts. I was changing it quite a bit so I didn't want it to be
tracked, but it's probably a good idea to have an example dir that comes
with files for the first time user.

**Comment by paulboone** [comment]
Ok, awesome. We should maybe make those into tests at some point!

**Pull request merged**